torch/csrc/functorch/init.cpp

```diff
@@ -4,25 +4,28 @@
 // This source code is licensed under the BSD-style license found in the
 // LICENSE file in the root directory of this source tree.
 
-#include <torch/extension.h>
-#include <ATen/WrapDimUtils.h>
 #include <ATen/FunctionalTensorWrapper.h>
+#include <ATen/WrapDimUtils.h>
+#include <torch/extension.h>
 
-#include <ATen/functorch/TensorWrapper.h>
-#include <ATen/functorch/DynamicLayer.h>
+#include <ATen/functorch/BatchRulesHelper.h>
+#include <ATen/functorch/BatchedFallback.h>
 #include <ATen/functorch/BatchedTensorImpl.h>
+#include <ATen/functorch/DynamicLayer.h>
 #include <ATen/functorch/LegacyVmapTransforms.h>
-#include <ATen/functorch/BatchedFallback.h>
-#include <ATen/functorch/BatchRulesHelper.h>
 #include <ATen/functorch/PlumbingHelper.h>
-#include <torch/csrc/functorch/CompileCache.h>
+#include <ATen/functorch/TensorWrapper.h>
 #include <c10/core/AutogradState.h>
 #include <functorch/csrc/dim/dim.h>
+#include <torch/csrc/functorch/CompileCache.h>
 
 // This file contains functorch's Python bindings.
 
-namespace at {
+namespace torch {
 namespace functorch {
+namespace impl {
+
+using namespace at::functorch;
```

Comment:
This file was moved, clang-format'ed, and namespaces were changed. The other changes are mentioned as inline comments on this PR.